Code/Tools/AssetProcessor/native/tests/PathDependencyManagerTests.cpp

```diff
@@ -350,6 +349,8 @@ namespace UnitTests
         AzToolsFramework::AssetDatabase::ProductDatabaseEntry m_product1, m_product2, m_product4;
         AzToolsFramework::AssetDatabase::ProductDependencyDatabaseEntryContainer m_dependencies;
 
+        void TestBody() override {}
```

Comment:
What is with the addition of the TestBody function?

Reply:
I kept getting confused by the inheritance structure of the previous class layout. I originally changed `PathDependencyBenchmarks` to now inherit from `LeakDetectionFixture`, which inherits from `testing::Test`, which was not in the inheritance chain before. But then when it is `new`'d:

https://github.com/o3de/o3de/blob/development/Code/Tools/AssetProcessor/native/tests/PathDependencyManagerTests.cpp#L465

it is missing an implementation for `TestBody()`, so that doesn't work. When I changed `PathDependencyBenchmarks` to inherit from `LeakDetectionBase` instead, then the `PathDependencyTestValidation` class was malformed, because I had removed the `testing::Test` base and forgot to add it back:

https://github.com/o3de/o3de/blob/development/Code/Tools/AssetProcessor/native/tests/PathDependencyManagerTests.cpp#L492-L495

My initial reaction was to just add an empty `TestBody()` method, but that's all fixed in the latest commit.